src/box/lua/load_cfg.lua

```diff
@@ -309,6 +313,18 @@ local function check_replicaset_uuid()
     end
 end
 
+local function load_wal_ext()
+    local wal_ext_cfg = box.cfg.wal_ext
+    local has_wal_ext, wal_ext = pcall(require, 'wal_ext')
```

Comment:
It's wrong to put it in a require-able module, because it should be invisible to users. Put it in `box.internal` instead.